File: ejemplos-clases-constructores/Ejemplo1/ejercicio1/src/ejercicio1/Jugadores.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ejercicio1;

/**
 *
 * @author USUARIO
 */
public class Jugadores {

    private String nombre;
    private String apellidos;

    public Jugadores(String x, String c) {

    }

    public void establecerNombre(String x) {
        nombre = x;
    }

    public void establecerApellido(String c) {
        apellidos = c;
    }

    public String obtenerNobre() {
        return nombre;
    }

    public String obtenerApellidos() {
        return apellidos;
    }
}

File: ejemplos-objetos/Ejemplo2/jugadores/jugadores.py
# Created on : 29 may 2024, 21:42:43
# Author     : USUARIO


# This is a sample Python script.

# Press Shift+F6 to execute it or replace it with your code.



class Jugadores:

    def __init__(self, x, c):
        self.nombre = x
        self.apellidos = c

    def establecer_nombre(self, x):
        self.nombre = x

    def establecer_apellido(self, c):
        self.apellidos = c

    def obtener_nombre(self):
        return self.nombre

    def obtener_apellidos(self):
        return self.apellidos